content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+यह पेज React के दस्तावेज और उससे सम्बंधित साधनो का ओवरव्यू है |
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** एक जावास्क्रिप्ट लाइब्रेरी है जो उपयोगकर्ता को इंटरफेसेस बनाने में काम आती है | [हमारे होमपेज](/) पर या [इस टुटोरिअल](/tutorial/tutorial.html) पर सीखें की React किस चीज़ के बारे में है |
```

Comment:
would be better worded as उपयोगकर्ता इंटरफेस के निर्माण के लिए एक जावास्क्रिप्ट लाइब्रेरी है

Reply:
[हमारे होमपेज](/) पर या [इस टुटोरिअल](/tutorial/tutorial.html) पर सीखें की React किस चीज़ के बारे में है | -> change to "react ke baare main jaane ke liye humare homage par ya tutorial ko dekhe"